### PR TITLE
bump nheko-git

### PR DESCRIPTION
**Pull request by Sazu-bit**
Adding nheko 0.10.2, should be a straight rebuild. Couldn't find any information how much detail is required.

### CODE REVIEW

**Comment by dr460nf1r3** [comment]
I'm not sure how this could be helpful? This file just controls how packages `pkgrel` is bumped due to rebuilds etc. If you want to request `nheko`, please open an issue at the packages repo. :)

File: PKGREL_BUMPS [hunk 1]
```diff
@@ -781,6 +781,7 @@ nginx-mainline-mod-echo 0.61-37 1
 nginx-mainline-mod-fancyindex 0.5.2-3 1
 nginx-mainline-mod-headers-more 0.34-1 1
 nginx-mainline-mod-ndk 0.3.2-1 1
+nheko 0.10.2
```
Comment (PedroHLC) [inline PKGREL_BUMPS]:
```suggestion
nheko-git 0.10.2.r155.g40a7d58c-1 1
```

**Review by PedroHLC — CHANGES_REQUESTED** [review]

**Review by PedroHLC — APPROVED** [review]

**Comment by PedroHLC** [comment]
I'm sorry @Sazu-bit, @dr460nf1r3 went ahead and pushed it directly to master.

Closed with: 04aab88d35ca70d5cba277c1d747624c8db36a12.

**Comment by dr460nf1r3** [comment]
Already done in https://github.com/chaotic-aur/interfere/commit/04aab88d35ca70d5cba277c1d747624c8db36a12, though thanks for the effort :)

**Pull request closed without merging**